thumbor/loaders/file_loader.py

```diff
@@ -39,18 +39,19 @@ def load(context, path, callback):
     if not exists(file_path):
         file_path = unquote(file_path)
 
-    if exists(file_path):
-        with open(file_path, 'rb') as f:
-            stats = fstat(f.fileno())
-
-            result.successful = True
-            result.buffer = f.read()
-
-            result.metadata.update(
-                size=stats.st_size,
-                updated_at=datetime.utcfromtimestamp(stats.st_mtime))
-    else:
-        result.error = LoaderResult.ERROR_NOT_FOUND
+    try:
+        result.successful = exists(file_path)
+        if result.successful:
+            with open(file_path, 'rb') as f:
+                stats = fstat(f.fileno())
+
+                result.buffer = f.read()
+
+                result.metadata.update(
+                    size=stats.st_size,
+                    updated_at=datetime.utcfromtimestamp(stats.st_mtime))
+        else:
+            result.error = LoaderResult.ERROR_NOT_FOUND
+    except Exception:
         result.successful = False
```

Comment:
Sorry to keep asking stuff, but I don't think this should be a silent error. We should log the error (thumbor logging) and also set result.exception to the exception that happened, don't you think?

Reply:
I'm agree with you. Fixed this.